docs/policy-only.md

```diff
@@ -0,0 +1,166 @@
+# Running Antrea In Policy Only Mode
+
+Antrea supports chaining with routed CNI implementations such as EKS CNI. In this mode, Antrea
+enforces Kubernetes NetworkPolicy, and delegates Pod IP management and network connectivity to the
+primary CNI.
+
+## Design
+
+Antrea is designed to work as NetworkPolicy plug-in for the routed CNIs. For as long as a CNI
+implementation fits into this model, Antrea may be inserted to enforce NetworkPolicy in that
+CNI's environment.
+
+<img src="/docs/assets/policy-only-cni.svg" width="600" alt="Antrea Switched CNI">
+
+The above diagram depicts a routed CNI network topology on the left, and what it looks like 
+after Antrea inserts the OVS bridge into the data path.
+
+The diagram on the left illustrates a routed CNI network topology such as AWS EKS.
+In this topology a Pod connects to the host network via a
+point-to-point(PtP) like device, such as (but not limited to) a veth-pair. On the host network, a
+host route with corresponding Pod's IP address as destination is created on each PtP device. Within
+each Pod, routes are configured to ensure all outgoing traffic is sent over this PtP device, and
+incoming traffic is received on this PtP device. This is a spoke-and-hub model, where to/from Pod
+traffic, even within the same worker Node must traverse first to the host network and being
+routed by it.
+
+When a Pod is instantiated, the container runtime first calls the primary CNI to configure Pod's
+IP, route table, DNS etc, and then connects Pod to host network with a PtP device such as a 
+veth-pair. When Antrea is chained with this primary CNI, container runtime then calls
+Antrea, and the Antrea agent attaches Pod's PtP device to the OVS bridge, and moves the host
+route to the Pod to local host gateway(``gw0``) interface from the PtP device. This is
```

Comment:
I think I asked before and it's a pretty minor point, but why do you use double backticks (``gw0``) instead of single ones (`gw0`)? Is there a difference in Github markdown?

Reply:
No, I think they are say, just habit.